src/status_im/accounts/login/core.cljs

```diff
@@ -273,13 +274,14 @@
                 (navigation/navigate-to-cofx :enter-pin-login nil)))))
 
 (fx/defn get-user-password
-  [_ address]
+  [_ address biometric-auth?]
```

Comment:
there is no need to pass it here, you can retrieve it from db in `:accounts.login.callback/get-user-password-success` event

Reply:
ah yes, right, was like that before introducing the profile setting...

Reply:
in any case i have to pass address to `:accounts.login.callback/get-user-password-success` to be able to lookup in db.